syz-fuzzer/proc.go

```diff
@@ -183,11 +184,45 @@ func (proc *Proc) triageInput(item *WorkTriage) {
 	}
 }
 
+func reexecutionSuccess(info *ipc.ProgInfo, oldInfo *ipc.CallInfo, call int) bool {
+	if info == nil || len(info.Calls) == 0 {
+		return false
+	}
+	if call != -1 {
+		if len(info.Calls[call].Signal) == 0 {
+			return false
+		}
+		// Don't minimize calls from successful to unsuccessful.
+		// Successful calls are much more valuable.
+		if oldInfo.Errno == 0 && info.Calls[call].Errno != 0 {
+			return false
+		}
+	}
+	if len(info.Extra.Signal) == 0 {
+		return false
+	}
+	return true
+}
+
+func getSignalAndCover(p *prog.Prog, info *ipc.ProgInfo, call int) (signal.Signal, []uint32) {
+	var inf *ipc.CallInfo
```

Comment:
2 lines shorter and no need to type the type:
```
	inf := &info.Extra
	if call != -1 {
		inf = &info.Calls[call]
	}
```

Reply:
Done.